internal/runners/export/docs/docs.md.tpl

```diff
@@ -5,14 +5,24 @@
 
 ## {{.Group.String}}{{end}}
 
-### {{.NameRecursive}}
+### {{.NameRecursive}} {{ if .Unstable }}(unstable) {{ end }}
```

Comment:
I've added a new AC that I had forgotten to add after our original discussions:

> Unstable commands are grouped together below stable commands

That's my bad, but we will want to still address it for this PR.